docs/src/tutorial/genomic-surveillance.rst

```diff
@@ -0,0 +1,177 @@
+4. Select **Download** > **Input for the Augur pipeline** > **Download**.
+5. Download/move the ``.tar`` file into the ``ncov/data/`` directory.
+6. Extract by opening the downloaded ``.tar`` file in your file explorer. It contains two files: one ending with ``.metadata.tsv`` and another with ``.sequences.fasta``.
+7.  Rename the files as appropriate, for example ``idaho.metadata.tsv`` and ``idaho.sequences.fasta``.
+
+Run the workflow
+----------------
+
+From within the ``ncov/`` directory, run the ``ncov`` workflow using a pre-written ``--configfile``:
+
+.. code:: text
+
+   nextstrain build . --cores all --configfile ncov-tutorial/genomic-surveillance.yaml
+
+Break down the command
+~~~~~~~~~~~~~~~~~~~~~~
+
+The workflow can take several minutes to run. While it is running, you can investigate the contents of ``genomic-surveillance.yaml`` (comments excluded):
+
+.. code-block:: yaml
+
+   inputs:
+     - name: reference_data
+       metadata: https://data.nextstrain.org/files/ncov/open/reference/metadata.tsv.xz
+       aligned: https://data.nextstrain.org/files/ncov/open/reference/aligned.fasta.xz
+     - name: custom_data
+       metadata: data/idaho.metadata.tsv
+       sequences: data/idaho.sequences.fasta
+     - name: background_data
+       metadata: https://data.nextstrain.org/files/ncov/open/north-america/metadata.tsv.xz
+       aligned: https://data.nextstrain.org/files/ncov/open/north-america/aligned.fasta.xz
+
+   refine:
+     root: "Wuhan-Hu-1/2019"
+
+   builds:
+     idaho:
+       title: "Idaho-specific genomic surveillance build"
+       subsampling_scheme: idaho_scheme
+       auspice_config: ncov-tutorial/auspice-config-custom-data.json
+
+   subsampling:
+     idaho_scheme:
+       custom_sample:
+         query: --query "(custom_data == 'yes')"
+         max_sequences: 5000
+       usa_context:
+         query: --query "(custom_data != 'yes') & (country == 'USA')"
+         max_sequences: 1000
+         group_by: division year month
+         priorities:
+           type: proximity
+           focus: custom_sample
+       global_context:
+         query: --query "(custom_data != 'yes')"
+         max_sequences: 1000
+         priorities:
+           type: proximity
+           focus: custom_sample
+
+This is similar to the previous file. Differences are outlined below, broken down per configuration section.
+
+inputs
+******
+
+1. The file paths in the second input are changed to ``idaho.metadata.tsv`` and ``idaho.sequences.fasta``.
+2. There is an additional input ``background_data`` for a regional North America dataset built by the Nextstrain team, for additional context.
+
+builds
+******
+
+The output dataset is renamed ``idaho``, representative of the new custom data in the second input.
+
+1. The title is updated.
+2. There is a new entry ``subsampling_scheme: idaho_scheme``. This is described in the following section.
```

Comment:
Perhaps a bit more on the _why_. E.g. "Without this the dataset would use all the provided data, which in this case is many thousand sequences."

Reply:
Done: 95d9f372

I reworded it a bit:

> Without this, the output dataset would use all the provided data, which in this case is thousands of sequences that are often disproportionally representative of the underlying population.